stardis/plasma/base.py

```diff
@@ -172,13 +170,15 @@ def create_splasma(marcs_model_fv, marcs_abundances_all, atom_data):
     # plasma_modules.remove(tardis.plasma.properties.general.SelectedAtoms)
     # plasma_modules.remove(tardis.plasma.properties.plasma_input.Density)
 
-    splasma = BasePlasma(
+    fv_geometry = stellar_model.fv_geometry
```

Comment:
Just FYI (no need to change if you like it this way), we can directly use object attribute below without needing to copy them in a local variable esp if it's similarly named.